src/main/java/seedu/duke/constants/ErrorMessage.java

```diff
@@ -5,7 +5,8 @@
  */
 public interface ErrorMessage {
     String INVALID_ADD_COMMAND = "Please check that you have correctly provided the currency and amount";
-    String INVALID_AMOUNT_TO_ADD_OR_WITHDRAW = "Please provide a value greater than or equal to 0.01";
+    String INVALID_TOO_SMALL_AMOUNT_TO_ADD_OR_WITHDRAW = "Please provide a value greater than or equal to 0.01";
```

Comment:
I added a similar error message for show-rate when the number is outside the range [0.01, 1b]. If mine ends up being merged second I can change to standardize.